rainier-core/src/main/scala/com/stripe/rainier/compute/Real.scala

```diff
@@ -117,7 +117,10 @@ object Real {
   val negInfinity: Real = NegInfinity
 }
 
-final private[rainier] case class Constant(value: BigDecimal) extends Real
+final private[rainier] case class Constant(value: BigDecimal) extends Real {
+  val toDouble = value.toDouble
```

Comment:
Is there any reasonable case where this is never evaluated such that we're better off making it lazy?

Reply:
Very good point. That is a common case and I'll make it lazy.